jenkins_node_scanner.py

```diff
@@ -100,9 +100,12 @@ def get_args():
     )
     parser.add_argument(
         '--target-port',
-        default=9100,
+        action='append',
+        default=[],
+        required=True,
```

Comment:
Passing `metavar='target_ports',`  here would allow to write `for port in args.target_ports` on line 233.

Reply:
I don't really understand this comment. I'm not familiar with `metavar`, so I looked up [the documentation](https://docs.python.org/dev/library/argparse.html#metavar) and found this:

> Note that metavar only changes the displayed name - the name of the attribute on the parse_args() object is still determined by the dest value.

I don't think this does what you think it does...

Also, line 233 is a blank line. Did you mean line 245? So in other words:

```python
node_info = [{
    'labels': {
        'jenkins_master': args.url,
        'node': node['name'],
    },
    'targets': ['%s:%d' % (ip_addr, port) for port in args.target_ports  # <-- here!
                if is_port_open(ip_addr, port)],
} for node, ip_addr in node_ips]
```

If that's what you meant, then doesn't work. It throws `AttributeError: 'Namespace' object has no attribute 'target_ports'`.

Reply:
Oh yeah, I meant `dest`. Sorry for the confusion.